### DISCUSSION TITLE
Bi-directional all-window search (current tab page!)?

### DISCUSSION THREAD

**Discussion by Maltimore** [ggandor/lightspeed.nvim#133] [Ideas]
Hi!
Lightspeed is awesome, and I can see from discussions here and on Reddit how much thought you put into it.
I know you are/were opposed to this, but I was wondering if, for extremely cursor-unaware people like me, we could get a setting that searches absolutely everywhere (on the current tab page)? All windows, in all directions. I'd like to map that to `s` and never think about anything else ;)
Edit: I see now that you raised an issue about this here https://github.com/ggandor/lightspeed.nvim/issues/130
However, in that issue there isn't talk yet of a full all-window search, as I think it still only searches the other window(s)?

**Comment by ggandor** [comment]
Well, you can do this in Leap now (https://github.com/ggandor/leap.nvim#search-mode-tweaks). It would be good to refactor Lightspeed too, but I don't know when I (or someone) will have the time for all the catching up.

**Reply by Maltimore** [reply]
Ah, wonderful!
I'm more than happy to switch to Leap from Lightspeed. From the Readme, Leap sounds almost exactly the same as  Lightspeed. Is there some kind of Lightspeed vs. Leap comparison?
Either way, thanks a lot for these great plugins. I use Lightspeed all the time.

**Reply by ggandor** [reply]
Most importantly: no shortcuts, no jump-to-unique-chars, and the f/t motions are not part of the plugin. In exchange: simpler UI, _much_ smaller visual footprint in the common case. It feels like using Sneak, but with the speed/efficiency of Lightspeed.

**Reply by Maltimore** [reply]
All of these sound really awesome! Thanks. I'll definitely make the switch.
